.github/workflows/test-matrix-2-levels.yml

```diff
@@ -0,0 +1,57 @@
+name: Test atmos-affected-stacks with 2 levels of nested matrices
+on:
+#  # Uncomment when test added first time to register workflow and comment it back after workflow would be registered
+#  #
+#  # Added pull_request to register workflow from the PR.
+#  # Read more https://stackoverflow.com/questions/63362126/github-actions-how-to-run-a-workflow-created-on-a-non-master-branch-from-the-wo
+#  pull_request: {}
+  workflow_dispatch: {}
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup
+        run: echo "Do setup"
```

Comment:
whats this job doing?

Reply:
It is testing the action

Reply:
it's not clear from the name of the step or from echoing a string why "setup" is necessary. but it's not important. resolved